uavcan/equipment/air_data/1027.RawAirData.uavcan

```diff
@@ -0,0 +1,29 @@
+#
+# Raw Air Data.
+#
+
+#
+# Heater State
+# 
+uint8 HEATER_UNAVAILABLE    = 0
+uint8 HEATER_WORKING        = 1
+uint8 HEATER_OVERCURRENT    = 2
+uint8 HEATER_OPENCIRCUIT    = 3
+uint8 heater_state
+
+#
+# Pressure Data
+#
+float32 static_pressure                 # Pascal
+float32 differential_pressure           # Pascal
+
+#
+# Temperature Data
+#
+float16 sensor_temperature              # Kelvin
+float16 static_air_temperature          # Kelvin
+float16 pitot_temperature               # Kelvin
+
+float16[<=25] covariance                # Pascal^2 for pressure variance and covariance
```

Comment:
@EShamaev Could you please list what variances we need here? Given the uncertain nature of OAT measurements, it might be useful to provide variance estimates at least for that?

@bugobliterator The ordering of the diagonal elements in the covariance matrix should be specified explicitly. Typically it would match the ordering of the preceding fields.